package.json

```diff
@@ -23,15 +23,15 @@
     "test:browser:ci": "npm run install:puppeteer && npm run test:browser",
     "install:puppeteer": "npm install --save-dev jest-puppeteer puppeteer @types/puppeteer",
     "prepare": "node ./scripts/tsc.js && node ./scripts/install.js",
-    "install": "node ./scripts/install.js"
+    "install": "node ./scripts/install.js --rebuild"
```

Comment:
is this necessary?

Reply:
no, will remove. i was experimenting with something and forgot to remove.